### PR TITLE
feat: add learn more link for `anonklub` project

### PR DESCRIPTION
**Pull request by sripwoud**
Anonklub is already archived.  
I am just adding a link to our [closing blog post](https://mirror.xyz/privacy-scaling-explorations.eth/7VTKFVR4PM75WtNnBzuQSBZW-UYoJOsnzBBQmB9MWbY).

|Before|After|
|---|---|
|image|image|

### CODE REVIEW

File: data/projects/mopro.ts [hunk 1]
```diff
@@ -48,7 +48,12 @@ export const mopro: ProjectInterface = {
   tags: {
     keywords: ["Mobile", "Client", "iOS", "Android"],
     themes: ["build", "play"],
-    types: ["Legos/dev tools", "Lego sets/toolkits", "Infrastructure/protocol", "Plugin",],
+    types: [
+      "Legos/dev tools",
+      "Lego sets/toolkits",
+      "Infrastructure/protocol",
+      "Plugin",
+    ],
```
Comment (sripwoud) [inline data/projects/mopro.ts:56]: unrelated: formatted automatically with `yarn format:write`

**Comment by samajammin** [comment]
Thanks @sripwoud!

What do you think about adding a link right in this warning text:
Screenshot 2024-11-01 at 00 23 56

Something like

> This project has been sunset and is not actively worked on anymore. [Read here for details on our learnings](https://mirror.xyz/privacy-scaling-explorations.eth/7VTKFVR4PM75WtNnBzuQSBZW-UYoJOsnzBBQmB9MWbY).

**Comment by sripwoud** [comment]
> Thanks @sripwoud!
> 
> 
> 
> What do you think about adding a link right in this warning text:
> 
> Screenshot 2024-11-01 at 00 23 56
> 
> 
> 
> Something like
> 
> 
> 
> > This project has been sunset and is not actively worked on anymore. [Read here for details on our learnings](https://mirror.xyz/privacy-scaling-explorations.eth/7VTKFVR4PM75WtNnBzuQSBZW-UYoJOsnzBBQmB9MWbY).
> 
> 

I can make this update of course.
I am afk for a couple of hours 🛫 and will make the change then

**Comment by kalidiagne** [comment]
> Thanks @sripwoud!
> 
> What do you think about adding a link right in this warning text: Screenshot 2024-11-01 at 00 23 56
> 
> Something like
> 
> > This project has been sunset and is not actively worked on anymore. [Read here for details on our learnings](https://mirror.xyz/privacy-scaling-explorations.eth/7VTKFVR4PM75WtNnBzuQSBZW-UYoJOsnzBBQmB9MWbY).

This is a general message for all the project when is `inactive`, i will make it possible to add also this link (optional)

**Review by kalidiagne — APPROVED** [review]

**Comment by sripwoud** [comment]
> This is a general message for all the project when is inactive, i will make it possible to add also this link (optional)

if it is not possible to custom this message by adding a link in it, then we are good to merge?
@kalidiagne @samajammin

**Comment by kalidiagne** [comment]
> > This is a general message for all the project when is inactive, i will make it possible to add also this link (optional)
> 
> if it is not possible yet to custom this message by adding a link in it, then we are good to merge? @kalidiagne @samajammin

We can proceed with the merge. I’ll add the custom link later. Could you please share the link here? @sripwoud

**Pull request merged**